### PR TITLE
Code cleanup for projectiles

### PR DESCRIPTION
**Pull request by IrishPacks**
This cleans up the code for projectiles as well as I changed values of multipliers slightly. I've tested already and it works. If you want to put fishing rods in there, go ahead. I avoided it as I was unsure if I should have or not.

### CODE REVIEW

File: src/pocketmine/item/Egg.php [hunk 1]
```diff
@@ -21,10 +21,9 @@
 
 namespace pocketmine\item;
 
-class Egg extends Item{
+class Egg extends ItemProjectile {
```
Comment (JackNoordhuis) [inline src/pocketmine/item/Egg.php]: Inconsistant formating, not a good idea to mix the formating up

Comment (IrishPacks) [inline src/pocketmine/item/Egg.php]: should I remove the space?

**Comment by Muqsit** [comment]
EnderPearl?

**Comment by IrishPacks** [comment]
If you'd like me to add the enderpearl, then sure why not.

**Comment by Muqsit** [comment]
Please do 👍

**Pull request merged**